app/controllers/admin/configs_controller.rb

```diff
@@ -129,10 +129,26 @@ class ConfigsController < Admin::ApplicationController
         home_feed_minimum_score
       ].freeze
 
+    IMAGE_FIELDS =
+      %w[
+        main_social_image
+        logo_png
+        secondary_logo_url
+        campaign_sidebar_image
+        mascot_image_url
+        mascot_footer_image_url
+        onboarding_logo_image
+        onboarding_background_image
+        onboarding_taskcard_image
+      ].freeze
+
+    VALID_URL = %r{\A(http|https)://([/|.|\w|\s|-])*.[a-z]{2,5}(:[0-9]{1,5})?(/.*)?$}.freeze
```

Comment:
This should check that the URL is valid without strictly checking that the URL is a valid _image URL_ like I was previously doing -- I have removed `.(?:jpg|gif|png)` and have added a check to ensure that the last part of the URL is valid instead. How do we feel about this?

cc @rhymes and @Ridhwana

Reply:
@rhymes I remember you telling me in one of my PR's that we could parse the url and use `url.is_a?(URI::HTTP)`, like this https://github.com/forem/forem/blob/master/app/liquid_tags/reddit_tag.rb#L69-L73 instead of using a REGEX?

Reply:
@juliannatetreault I think it's fine, unfortunately there's no builtin way to validate URLs and `URI.parse()` is not super reliable by itself withouth checking at least the scheme. 

One final note: the regexp should end with `\z`, not `$`, the reason is because `$` matches end of line, not end of string. See https://rubystyle.guide/#caret-and-dollar-regexp

Reply:
Thank you for the feedback! I've made this change, so it should be good-to-go pending your review @rhymes 😄